docs/advanced/sa-column.md

```diff
@@ -0,0 +1,33 @@
+# SQLAlchemy Columns
+
+In some cases you may need more control over the columns generated by SQLModel, this can be done by using the `sa_column`, `sa_column_args`, and `sa_column_kwargs` arguments when creating the `Field` object.
+
+There are many use cases for this, but ones where this is particularity useful is when you want more advanced defaults for values than what is easy to implement with Pydantic, such `created_at` or `update_at` timestamps for rows.().
+
+## Columns for Timestamps
+
+Two ways of implementing `created_at` timestamps with Pydantic are [default factories](https://pydantic-docs.helpmanual.io/usage/models/#field-with-dynamic-default-value) and [validators](https://pydantic-docs.helpmanual.io/usage/validators/#validate-always), however there's no straightforward way to have an `update_at` timestamp.
```

Comment:
I think that an explicit example of Pydantic implementation (both like you did it #370) could be helpful for people preferring to use "all Pydantic" models.

Reply:
I've added a section with comments on the Pydantic implementation as well.